subsys/modem/backends/modem_backend_uart_isr.c

```diff
@@ -189,6 +240,23 @@ struct modem_pipe_api modem_backend_uart_isr_api = {
 	.close = modem_backend_uart_isr_close,
 };
 
+#if CONFIG_MODEM_STATS
+static void init_stats(struct modem_backend_uart *backend)
+{
+	char name[CONFIG_MODEM_STATS_BUFFER_NAME_SIZE];
+	uint32_t receive_buf_size;
+	uint32_t transmit_buf_size;
+
+	receive_buf_size = get_receive_buf_capacity(backend);
+	transmit_buf_size = get_transmit_buf_capacity(backend);
+
+	snprintk(name, sizeof(name), "%s_%s", backend->uart->name, "rx");
+	modem_stats_buffer_init(&backend->receive_buf_stats, name, receive_buf_size);
+	snprintk(name, sizeof(name), "%s_%s", backend->uart->name, "tx");
```

Comment:
Do those names fit in the default 16 bytes?

Reply:
depends on the board, I will increase it to 32 to be safe